Mathlib/Control/Random.lean

```diff
@@ -144,7 +148,7 @@ variable {m : Type* → Type*} {m₀ : Type → Type}
 variable [Monad m] [MonadLiftT (ST RealWorld) m₀] [ULiftable m₀ m]
 
 /--
-Computes a `RandT m α` using the global `stdGenRef` as RNG.
+Compute a `RandT m α` using the global `stdGenRef` as RNG.
```

Comment:
Maybe `Execute` is better here and below

Reply:
All done.